File: 프로그래머스/0/181844. 배열의 원소 삭제하기/배열의 원소 삭제하기.cs
using System;
using System.Collections.Generic;
public class Solution {
    public int[] solution(int[] arr, int[] delete_list) {
        List<int> list = new List<int>();
        for(int i = 0 ; i< arr.Length;i++)
        {
            list.Add(arr[i]);
            for(int j = 0; j < delete_list.Length; j++)
            {
                if(arr[i] == delete_list[j])
                {
                    list.Remove(arr[i]);
                    break;
                }
            }
        }
        
        int[] answer = list.ToArray();
        return answer;
    }
}

File: 프로그래머스/0/181844. 배열의 원소 삭제하기/배열의 원소 삭제하기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> solution(vector<int> arr, vector<int> delete_list) {
    vector<int> answer = arr;
    for(auto i : delete_list)
    {
        for(auto j : answer)
        {
            if(i == j)
            {
                auto new_end = remove(answer.begin(), answer.end(), i);
                answer.erase(new_end, answer.end());
            }
        }
    }
    
    return answer;
}